h2o-genmodel/src/main/java/hex/genmodel/algos/gbm/GbmMojoModel.java

```diff
@@ -1,15 +1,17 @@
 package hex.genmodel.algos.gbm;
 
 import hex.genmodel.GenModel;
+import hex.genmodel.algos.tree.SharedTreeGraph;
 import hex.genmodel.algos.tree.SharedTreeMojoModel;
+import hex.genmodel.algos.tree.TreeBackedMojoModel;
 import hex.genmodel.utils.DistributionFamily;
 
 import static hex.genmodel.utils.DistributionFamily.*;
 
 /**
  * "Gradient Boosting Machine" MojoModel
  */
-public final class GbmMojoModel extends SharedTreeMojoModel {
```

Comment:
same as DRF

Reply:
As stated above: Makes perfect sense. Moved to abstract predecessor. Removed from all the inheriting classes, including DrfMojoModel.